### PR TITLE
[BUGFIX release-1-13] Add deprecation options to some 1.13 deprecations

### PR DESCRIPTION
**Pull request by toddjordan**
I added deprecation options to some deprecation warnings I was commonly getting in going from 1.13 to 2.0.  This is probably not all the un-optioned deprecations, but they cover the ones I've seen so far.  This hopefully will help others going through the same process.

### CODE REVIEW

File: packages/ember-views/lib/views/view.js [hunk 1]
```diff
@@ -1345,15 +1345,21 @@ var View = CoreView.extend(
   scheduleRevalidate(node, label, manualRerender) {
     if (node && !this._dispatching && node.guid in this.env.renderedNodes) {
       if (manualRerender) {
-        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`);
+        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`,
+                        false,
+                        { id: 'ember-views.manual-parent-rerender', until: '3.0.0' });
```
Comment (toddjordan) [inline packages/ember-views/lib/views/view.js]: In the 2.0 release these were marked 3.0.0, even though in the text of the message it says its going away in 2.0.  Not sure what to put here.

**Comment by toddjordan** [comment]
A PR for issue https://github.com/emberjs/ember.js/issues/12478

File: packages/ember-views/lib/views/view.js (hunk 1 above)
Comment (mixonic) [inline packages/ember-views/lib/views/view.js]: it is already changed in master, and should be changed to 3.0 here https://github.com/emberjs/ember.js/blob/341a325a13fc685a10399de600586b6fb744e99f/packages/ember-views/lib/mixins/view_support.js#L682

Comment (mixonic) [inline packages/ember-views/lib/views/view.js]: Also right below

**Comment by mixonic** [comment]
Merged and I added the cleanup myself in https://github.com/emberjs/ember.js/commit/223ddd2a1076a78c2f7aaa8bde6c8f2f478304ea

Thanks @toddjordan !

**Comment by toddjordan** [comment]
Thanks @mixonic :+1:

**Pull request merged**